src/launchpad/service.py

```diff
@@ -93,9 +93,7 @@ def signal_handler(signum: int, frame) -> None:
 
     def is_healthy(self) -> bool:
         """Get overall service health status."""
-        is_server_healthy = self.server.is_healthy()
-        is_kafka_healthy = self.kafka.is_healthy()
-        return is_server_healthy and is_kafka_healthy
```

Comment:
Could you instead just change `kafka`'s `is_healthy` to always return true like `server.py` already does? That way we only have to touch one file and it's clear we are short-circuiting Kafka's health check.

Reply:
sure